### ISSUE TITLE
Automatic pre_commit_hook installation did not work

### ISSUE THREAD

**Issue by DariaKunoichi** [ejoffe/spr#94]
Hi, I tried .deb installation as well as in brew.
After running "git spr update" there was a problem with commit hook:
`git spr update
...
Add ticket from branch to commit message.................................Passed
Run StackedPR message hook...............................................Failed
- hook id: spr
- exit code: 1

Executable spr_commit_hook not found`

It was not linked to .git/hooks/commit_msg unfortunately. 
I had to find your older commit with instruction and link it manually.

**Comment by ejoffe** [comment]
glad you were able to link it manually, did everything work for you after that?

what operating system are you using? (I'm guessing osx)
what's the output of 'which spr_commit_hook' ?
also can you paste the whole output of the errored 'git spr update' call?

**Comment by DariaKunoichi** [comment]
Yes, after linking manually all worked well.
OS: Ubuntu 20.
Which spr_commit_hook depended on the installation method - I mentioned that I tried both brew and deb. So right now I the output is:
`/usr/local/bin/spr_commit_hook`.
Unfortunately I copied output only until this point:
`git spr update
Trim Trailing Whitespace.............................(no files to check)Skipped
Fix End of Files.....................................(no files to check)Skipped
Check Yaml...........................................(no files to check)Skipped
Check for added large files..........................(no files to check)Skipped
Check that executables have shebangs.................(no files to check)Skipped
Check that scripts with shebangs are executable......(no files to check)Skipped
Check JSON...........................................(no files to check)Skipped
Detect Private Key...................................(no files to check)Skipped
Mixed line ending....................................(no files to check)Skipped
Trim Trailing Whitespace.............................(no files to check)Skipped
Terraform fmt........................................(no files to check)Skipped
Terraform validate...................................(no files to check)Skipped
tflint...............................................(no files to check)Skipped
Shellcheck Bash Linter...............................(no files to check)Skipped
go-revive............................................(no files to check)Skipped
go-fmt...............................................(no files to check)Skipped
Check hooks apply to the repository..................(no files to check)Skipped
Check if CF pipelines pck setting has comment........(no files to check)Skipped
Add ticket from branch to commit message.................................Passed
Run StackedPR message hook...............................................Failed
- hook id: spr
- exit code: 1

Executable `spr_commit_hook` not found`
